STAR/draft-ietf-acme-star.md

```diff
@@ -516,6 +516,72 @@ setting it to true.
 
 If the server accepts the request, it MUST reflect the key in the Order.
 
+## Computing notBefore and notAfter of STAR Certificates
+
+We define the "nominal renewal date" as the point in time when a new short-term
+certificate for a given STAR Order is due.  It is a multiple of the Order's
+recurrent-certificate-validity which starts with the issuance of the
+first short-term certificate and it is upper bounded by the Order's
+recurrent-end-date ({{fignrd}}).
+
+~~~
+    rcv    - STAR Order's recurrent-certificate-validity
+    red    - STAR Order's recurrent-end-date
+    nrd[i] - nominal renewal date of the i-th STAR certificate
+
+
+                 .-rcv-.   .-rcv-.   .-rcv-.   ._rcv_.
+                /       \ /       \ /       \ /  red  \
+    -----------o---------o---------o---------o----X-------> t
+              nrd[0]    nrd[1]    nrd[2]    nrd[3]
+~~~
+{: #fignrd title="Nominal Renewal Date"}
+
+The rules to determine the notBefore and notAfter values of the i-th STAR
+certificate are as follows:
+
+~~~
+  notBefore = nrd[i] - predating
+  notAfter = min(nrd[i] + rcv, red)
+~~~
+
+where "predating" is the max between the (optional)
+recurrent-certificate-predate (rcp) and the amount of pre-dating that the
+server needs to add to make sure that all certificates being published are
+valid at the time of publication ({{fetching-certificates}}).  The server
+pre-dating is a fraction of rcv (1/n with n>=2).
+
+~~~
+  predating = max(rcp, rcv * 1/n)
+~~~
+
+### Example
+
+Given a server that intends to publish the next cert halfway through the
+lifetime of the previous one, and a STAR Order with the following settings:
+
+~~~
+     {
+       "recurrent-start-date": "2016-01-10T00:00:00Z",
+       "recurrent-end-date": "2016-01-20T00:00:00Z",	// 12d
+       "recurrent-certificate-validity": 345600,	// 4d
+       "recurrent-certificate-predate": 518400,		// 6d
+     }
+~~~
+
+The amount of pre-dating that needs to be subtracted from each nominal renewal
+date is 6 days -- computed as max(518400, 345600 * 1/2).
+
+The notBefore and notAfter of each short-term certificate is as follows:
```

Comment:
Edit: 

``
The "notBefore" and "notAfter" values of each short-term certificate are as follows:
``

Reply:
This has been already fixed in be84f6e